### PR TITLE
KOGITO-2719 KOGITO-3009 Consolidate tests using kogito-test-utils

### PR DESCRIPTION
**Pull request by cristianonicolai**
Depends on:
- https://github.com/kiegroup/kogito-apps/pull/344
- https://github.com/kiegroup/kogito-examples/pull/322

### CODE REVIEW

**Review by Sgitario — APPROVED** [review]

**Comment by cristianonicolai** [comment]
@tiagodolphine this also removes the job service integration tests

**Comment by Sgitario** [comment]
@cristianonicolai we're losing the tests of ProcessTimerIT (for spring and quarkus). This was meant to be done as part of KOGITO-2719 (restore the old suite of tests). 
Moreover, I was planning to get rid of this module as part of KOGITO-2985 (which I will close as duplicate as you already did it here). Are you ok with using KOGITO-2719 to restore the old suite of ITs for the ProcessTimerIT (for spring and quarkus) ?

**Review by radtriste — APPROVED** [review]

**Comment by cristianonicolai** [comment]
@Sgitario I updated so KOGITO-2719 is also included as part of this PR

**Comment by Sgitario** [comment]
> @Sgitario I updated so KOGITO-2719 is also included as part of this PR

@cristianonicolai but the old tests that were part of process timer examples have not been restored (I mean [these tests](https://github.com/kiegroup/kogito-examples/tree/0.9.x/process-timer-quarkus/src/test/java/org/acme/travels)). Do you think it's not necessary to restore them?

**Comment by cristianonicolai** [comment]
@Sgitario from what I understood these are now covered in kogito-apps repo, see https://github.com/kiegroup/kogito-apps/blob/master/integration-tests/integration-tests-common/src/main/java/org/kie/kogito/it/jobs/BaseProcessTimerIT.java#L33
@tiagodolphine can you confirm that?

**Review by evacchi — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -56,8 +56,8 @@
     <infinispan.starter.version>2.2.3.Final</infinispan.starter.version>
     <container.image.infinispan>quay.io/infinispan/server:${infinispan.version}</container.image.infinispan>
     <!-- keycloak -->
-    <container.image.keycloak.version>11.0.0</container.image.keycloak.version>
-    <container.image.keycloak>quay.io/keycloak/keycloak:${container.image.keycloak.version}</container.image.keycloak>
+    <keycloak.version>11.0.0</keycloak.version>
+    <container.image.keycloak>quay.io/keycloak/keycloak:${keycloak.version}</container.image.keycloak>
```
Comment (Sgitario) [inline pom.xml:60]: Same comment as in https://github.com/kiegroup/kogito-runtimes/pull/692#discussion_r469048216

**Review by tiagodolphine — APPROVED** [review]
LGTM

**Pull request merged**